src/components/SelectCol.js

```diff
@@ -63,8 +63,11 @@ class SelectCol extends React.Component {
             <div id = "selectColumns" ref={ node => this.node = node }>
                 <button className = "btn btn-light selectColButton"  onClick = { this.onButtonClick }  title = "Select Languages">
                     <i className="fas fa-tasks"/>
+
```

Comment:
why the button didn't have the dark mode??? I pulled down and found the two buttons (SelectCol and Toggle10andall are still white not dark mode.) 
![image](https://user-images.githubusercontent.com/49899381/97647422-29448780-1a8d-11eb-84c2-606fa5fc35b8.png)